### PR TITLE
增加中文维基百科用户贡献数统计

### PR DESCRIPTION
**Pull request by scvoet**

### CODE REVIEW

**Comment by spencerwooo** [comment]
Thanks! I'll look into this when I have time later this week.

**Review by spencerwooo — APPROVED** [review]
👍

File: src/utils/handlers/wikipedia-zh.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    const response = await fetchWikipediazhStats(name)
+    const stats = await response.json()
+    const res = {
+      source: 'wikiepdia-zh',
+      subs: 0,
+      failed: false,
+      failedMsg: '',
+    }
+  
+    if (response.stats.query.users.0.missing = '') {
```
Comment (spencerwooo) [inline src/utils/handlers/wikipedia-zh.js:29]: This doesn't seem right.

File: src/utils/handlers/wikipedia-zh.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      // Wikipedia-zh user not found
+      res.failed = true
+      res.subs = 0
+      res.failedMsg = 'Wikipedia-zh user not found'
+    } else {
+      // Wikipedia-zh user found
+      res.subs = stats.query.users.0.editcount
```
Comment (spencerwooo) [inline src/utils/handlers/wikipedia-zh.js:36]: Neither does this.

**Review by spencerwooo — COMMENTED** [review]
Sorry I was in a rush in merging PRs today. Could you give me an example for a valid Wikipedia username that I can test with the API you provided here?

```
https://zh.wikipedia.org/w/api.php?action=query&list=users&ususers=${name}&usprop=editcount&format=json
```

Just a username I can substitute `${name}` with.

**Comment by scvoet** [comment]
> 
> 
> Sorry I was in a rush in merging PRs today. Could you give me an example for a valid Wikipedia username that I can test with the API you provided here?
> 
> ```
> https://zh.wikipedia.org/w/api.php?action=query&list=users&ususers=${name}&usprop=editcount&format=json
> ```
> 
> Just a username I can substitute `${name}` with.

Sure, and `scvoet` is available.

**Comment by spencerwooo** [comment]
@scvoet I corrected and cleaned up the code for you on PR #25 if you would like to have a look.

In the meantime, could you provide a little bit more for the documentation part of Substats - Wikipedia (zh)? Which is at [API details - Wikipedia (zh)](https://substats.spencerwoo.com/api.html#wikipedia-zh). The `queryKey` part is especially missing key information as I myself is not familiar with the API.

**Comment by scvoet** [comment]
> 
> 
> @scvoet I corrected and cleaned up the code for you on PR #25 if you would like to have a look.
> 
> In the meantime, could you provide a little bit more for the documentation part of Substats - Wikipedia (zh)? Which is at [API details - Wikipedia (zh)](https://substats.spencerwoo.com/api.html#wikipedia-zh). The `queryKey` part is especially missing key information as I myself is not familiar with the API.

Of course, I will finish the document today.

**Pull request merged**